### DISCUSSION TITLE
Gift wrap on checkout page

### DISCUSSION THREAD

**Discussion by tjain90** [inspireui/support#5077] [Working on Features]
Hi Team,
Gift wrap is something that any woocomerce store require. It's great if you can integrate https://docs.woocommerce.com/document/woocommerce-checkout-add-ons/

This will help customer to enter gift wrap, gift wrap cost and message as well. Also if anytime we need additional item like chocolates or something else with full cart then we can do that.

**Comment by osman-kocak** [comment]
we need that

**Comment by antimimin** [comment]
If you use webview checkout. The plugin will work.

**Reply by osman-kocak** [reply]
ı want show on native side

**Reply by tjain90** [reply]
Webview discard the amazing experience of native app

**Comment by tjain90** [comment]
Hi Team, Can you please have a look on this

**Reply by tryabp** [reply]
checkout add ons is must have feature. i hope fluxstore will add that in future soon

**Comment by tryabp** [comment]
Hello, as i seen green mark on this topic. I just want to know if all add on types will be supported or not? 
I am asking in context of checkout files upload feature, current file upload feature in product addons is not suitable for pharmacy business where customer adds multiple products to cart and then only 1 or 2 files prescription on cart or checkout page. 

Please see what you can do to implement this [Checkout Addons Plugin](https://woocommerce.com/document/woocommerce-checkout-add-ons/) where we can take fluxstore checkout experience to new level.

**Comment by tryabp** [comment]
any update on this feature request @minhcasi
